### DISCUSSION TITLE
【求助】鱼要是会抽烟，那是怎么抽的？？求各位绳匠解答！！

### DISCUSSION THREAD

**Discussion by halin1117** [share121/inter-knot#2345] [其他]
![IMG_20240816_003417](https://github.com/user-attachments/assets/112412ee-1cae-4a68-a3c1-27af6ab65bbc)
![IMG_20240816_003400](https://github.com/user-attachments/assets/875519fc-537e-47bd-ae1f-ed0917b50162)

**Comment by jianjue1** [comment]
应如头图所示，毕竟吸烟使用的是呼吸系统，人类的食道和气管同样是分开的，只是共用鼻咽<img src="https://bbs-static.miyoushe.com/static/2023/04/28/0c624a8915dd9b24093cf630bded5748_7288657913684710277.png" alt="米仔_拜年" width="75" >

**Comment by Genshin-Neuvillette** [comment]
鱼能吸收尼古丁吗
